src/styles/blocks/header.scss

```diff
@@ -0,0 +1,122 @@
+.header {
+  @include paddingContainer();
+  background-image: url(../images/header-logo.png);
+  background-size: cover;
+  background-position: 90%;
+  background-repeat: no-repeat;
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  box-sizing: border-box;
+
+  &__top {
+    @include retreatTopHeader();
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    @include onTablet() {
+      padding-block: 36px;
+    }
+  }
+
+  &__controls {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    gap: 24px;
+
+    @include onTablet() {
+      gap: 32px;
+    }
+  }
+
+  &__wrapper-phone {
+    display: flex;
+    align-items: center;
+    gap: 20px;
+
+    @include onDesktop() {
+      &:hover {
+        .header__phone-number {
+          transform: scale(1);
+          transition: $transition;
+        }
+      }
+    }
+  }
+
+  &__phone-number {
+    display: none;
+
+    @include onTablet() {
+      color: $textColor;
+      font-size: 12px;
+      font-weight: 700;
+      line-height: 16px;
+      letter-spacing: 2px;
+      text-decoration: none;
+      transform: scale(0);
+      display: block;
+    }
+  }
+
+  &__icon {
+    width: 24px;
+    height: 24px;
+    display: block;
+    cursor: pointer;
+
+    &--phone {
+      background-image: url(../images/icons/phone.svg);
+      background-size: cover;
+    }
+
+    &--menu {
+      background-image: url(../images/icons/burger-menu.svg);
+      background-size: cover;
+      transition: $transition;
+
+      @include onDesktop() {
+        &:hover {
+          background-image: url(../images/icons/burger-menu-hover.svg);
+        }
+      }
+    }
+
+    @include onTablet() {
+      width: 32px;
+      height: 32px;
+    }
+  }
+
+  &__title {
+    margin: 0;
+    margin-bottom: 40%;
+    text-align: center;
+    font-size: 36px;
+    font-weight: 700;
+    line-height: 100%;
+
+    @include onTablet() {
+      font-size: 64px;
+    }
+
+    @include onDesktop() {
+      font-size: 80px;
+      margin-bottom: 20%;
+    }
+  }
+}
+
+.logo {
```

Comment:
make it as a header element

Reply:
I reuse this styles in few places and just moved this block to global file. Or you are toking about something another?